### ISSUE TITLE
add bower package

### ISSUE THREAD

**Issue by ghigt** [hamdouni/md-date#2]
Could be better to have a bower package.
http://bower.io/docs/creating-packages/

**Comment by hamdouni** [comment]
now we can do
bower install md-date

**Comment by ghigt** [comment]
We have conflicts with angular-material > 0.8 (breaks the website), and during the install of `md-date`, it forces me to choose:

```
Unable to find a suitable version for angular-material, please choose one:
    1) angular-material#~0.9 which resolved to 0.9.4 and is required by rgc
    2) angular-material#~0.10.0 which resolved to 0.10.0 and is required by md-date#b623c9c34e
```

**Comment by hamdouni** [comment]
fixed to use 0.9.x
